magicctapipe/scripts/lst1_magic/semi_automatic_scripts/dl1_production.py

```diff
@@ -455,98 +230,49 @@ def main():
-                if len(list_of_MC) < 2:
-                    logger.warning(
-                        f"No bash script has been produced for processing {particle}"
-                    )
-                else:
-                    launch_jobs_MC = f"linking=$(sbatch --parsable linking_MC_{particle}_paths.sh) && running=$(sbatch --parsable --dependency=afterany:$linking linking_MC_{particle}_paths_r.sh)"
-                    os.system(launch_jobs_MC)
-            # Here we do the MC DL0 to DL1 conversion:
 
     for source_name in source_list:
-        if (
-            (args.analysis_type == "onlyMAGIC")
-            or (args.analysis_type == "doEverything")
-            or (NSB_match)
-        ):
-
-            MAGIC_runs_and_dates = f"{source_name}_MAGIC_runs.txt"
-            MAGIC_runs = np.genfromtxt(
-                MAGIC_runs_and_dates, dtype=str, delimiter=",", ndmin=2
-            )  # READ LIST OF DATES AND RUNS: format table where each line is like "2020_11_19,5093174"
-
-            # TODO: fix here above
-            print("*** Converting Calibrated into DL1 data ***")
-            print(f"Process name: {source_name}")
-            print(
-                f"To check the jobs submitted to the cluster, type: squeue -n {source_name}"
-            )
-
-            directories_generator_real(
-                str(target_dir), telescope_ids, MAGIC_runs, NSB_match, source_name
-            )  # Here we create all the necessary directories in the given workspace and collect the main directory of the target
-            config_file_gen(
-                target_dir, noise_value, NSB_match, source_name, config
-            )  # TODO: fix here
-
-            # Below we run the analysis on the MAGIC data
-
-            lists_and_bash_gen_MAGIC(
-                target_dir,
-                telescope_ids,
-                MAGIC_runs,
-                source_name,
-                env_name,
-                cluster,
-            )  # MAGIC real data
-            if (telescope_ids[-2] > 0) or (telescope_ids[-1] > 0):
-                list_of_MAGIC_runs = glob.glob(f"{source_name}_MAGIC-*.sh")
-                if len(list_of_MAGIC_runs) < 1:
-                    logger.warning(
-                        "No bash script has been produced. Please check the provided MAGIC_runs.txt and the MAGIC calibrated data"
-                    )
-                    continue
 
-                launch_jobs = f"linking=$(sbatch --parsable {source_name}_linking_MAGIC_data_paths.sh)"
-                for n, run in enumerate(list_of_MAGIC_runs):
-                    launch_jobs = f"{launch_jobs} && RES{n}=$(sbatch --parsable --dependency=afterany:$linking {run})"
+        MAGIC_runs_and_dates = f"{source_name}_MAGIC_runs.txt"
+        MAGIC_runs = np.genfromtxt(
+            MAGIC_runs_and_dates, dtype=str, delimiter=",", ndmin=2
+        )  # READ LIST OF DATES AND RUNS: format table where each line is like "2020_11_19,5093174"
+
+        # TODO: fix here above
```

Comment:
is this TODO still valid?
it sounds a bit cryptic

Reply:
I'm checking all the TODO because some of them are old leftovers. And some of them can be understood only by me 😅